modules/extensions-support/src/main/java/org/mule/runtime/module/extension/api/loader/AbstractJavaExtensionModelLoader.java

```diff
@@ -86,6 +88,7 @@
 public class AbstractJavaExtensionModelLoader extends ExtensionModelLoader {
 
   private static final boolean IGNORE_DISABLED = getProperty(DISABLE_SDK_IGNORE_COMPONENT) != null;
+  private final boolean ENABLE_POLLING_SOURCE_LIMIT = getProperty(ENABLE_SDK_POLLING_SOURCE_LIMIT) != null;
```

Comment:
This can be static too

Reply:
The tests rely on the value of the property changing. What I can try to do is find a way for the tests to add the parameter to the ExtensionLoadingContext, instead of doing this through the system prop.